src/univariate/continuous/chernoff.jl

```diff
@@ -0,0 +1,320 @@
+         1.0102911018761945e-26
+        -9.976759882474874e-29
+        ]
+    const b=[
+         0.6666666666666666
+         0.021164021164021163
+        -0.0007893341226674574
+         1.9520978781876193e-5
+        -3.184888134149933e-7
+         2.4964953625552273e-9
+         3.6216439304006735e-11
+        -1.874438299727797e-12
+         4.393984966701305e-14
+        -7.57237085924526e-16
+         1.063018673013022e-17
+        -1.26451831871265e-19
+         1.2954000902764143e-21
+        -1.1437790369170514e-23
+         8.543413293195206e-26
+        -5.05879944592705e-28
+        ]
+    const airyai_roots=[
+        -2.338107410459767
+        -4.08794944413097
+        -5.520559828095551
+        -6.786708090071759
+        -7.944133587120853
+        -9.02265085334098
+        -10.040174341558085
+        -11.008524303733262
+        -11.936015563236262
+        -12.828776752865757
+        -13.691489035210719
+        -14.527829951775335
+        -15.340755135977997
+        -16.132685156945772
+        -16.90563399742994
+        -17.66130010569706
+        ]
+    const airyai_prime=[
+         0.7012108227206906
+        -0.8031113696548534
+         0.865204025894141
+        -0.9108507370495821
+         0.9473357094415571
+        -0.9779228085695176
+         1.0043701226603126
+        -1.0277386888207862
+         1.0487206485881893
+        -1.0677938591574279
+         1.0853028313507
+        -1.1015045702774968
+         1.1165961779326556
+        -1.1307323104931881
+         1.1440366732735523
+        -1.156609849116565
+        ]
+
+    const cuberoottwo = cbrt(2.0)
+    const sqrthalfpi = sqrt(0.5*pi)
+    const sqrttwopi = sqrt(2.0*pi)
+
+    function p(y::Real)
+        if iszero(y)
+            return -sqrt(0.5*pi) 
+        end
+
+        (y > 0) || throw(DomainError(y, "argument must be positive"))
+
+        cnsty = y^(-1.5)
+        if (y <= 1.0)
+            return sum([(b[k]*cnsty - a[k]*sqrthalfpi)*y^(3*k) for k=1:length(a)])-sqrthalfpi
+        else
+            return sum([exp(cuberoottwo*a*y) for a in airyai_roots]) * 2 * sqrttwopi * exp(-y*y*y/6) - cnsty
+        end
+    end
+
+    function g(x::Real)
+        function g_one(y::Real)
+            return p(y) * exp(-0.5*y*(2*x+y)*(2*x+y))
+        end
+        function g_two(y::Real)
+            z = 2*x+y*y
+            return (z*y*y + 0.5 * z*z) * exp(-0.5*y*y*z*z)
+        end
+        if (x <= -1.0)
+            return cuberoottwo*cuberoottwo * exp(2*x*x*x/3.0) * sum([exp(-cuberoottwo*airyai_roots[k]*x) / airyai_prime[k] for k=1:length(airyai_roots)])
+        else
+            return 2*x - (quadgk(g_one, 0.0, Inf)[1] - 4*quadgk(g_two, 0.0, Inf)[1]) / sqrttwopi   # should perhaps combine integrals
+        end
+    end
+end
+
+_pdf(x::Real) = g(x)*g(-x)*0.5
+_cdf(x::Real) = (x < 0.0) ? _cdfbar(-x) : 0.5 + quadgk(_pdf,0.0,x)[1] 
+_cdfbar(x::Real) = (x < 0.0) ? _cdf(x) : quadgk(_pdf, x, Inf)[1]
+
+
+
+pdf(d::Chernoff, x::Real) = _pdf(x)
```

Comment:
you need to call `ChernoffComputations._pdf(x)` here, all functions and constants defined in the `ChernoffComputations` must be used this way